docs/markdown/Invoke-ScriptAnalyzer.md

```diff
@@ -219,7 +207,7 @@ Accept wildcard characters: False
 ```
 
 ### -CustomRulePath
-Adds the custom rules defined in the specified paths to the analysis.
+Uses only the custom rules defined in the specified paths to the analysis. To still use the built-in rules, additionally use the -IncludeDefaultRules switch.
```

Comment:
maybe just "add the -IncludeDefaultRules switch" instead of "additionally use the -IncludeDefaultRules switch" to not repeat word "use" ?